locales.software/1038-hu.cfos.txt

```diff
@@ -763,7 +763,7 @@ err_os_minxp = "Az ön operációs rendszere %?.\n\n%? futtatásához Windows XP
 err_os_sup = "%p hivatalosan csak a %? vagy magasabb verziót támogatja.\n\n"
              "Lehetséges, de általunk nincs tesztelve, hogy a %p elfut-e az ön %? operációs rendszerén.\n\n"
              "Biztos szeretné folytatni a telepítést?";
-err_uninst_driver = "Az eszközmeghajtó eltávolítása sikertelen.";
+err_uninst_driver = "Az eszközmeghajtó (driver) eltávolítása sikertelen.";
```

Comment:
Just post changes in that branch https://github.com/misibacsi/l10n/tree/patch-1

Regarding `driver` — okey. In some languages we're using similar thing (English naming near translated one). Just ensure that in case of you language it's indeed necessary. For example, try to Google it.